javascript/modules/common/src/encoding.ts

```diff
@@ -52,3 +52,12 @@ export function ensure0x(value: string): string {
 export function strip0x(value: string): string {
   return value.startsWith("0x") ? value.substring(2) : value;
 }
+
+export function hexToAscii(hex: string) {
+  var hex = hex.toString();
```

Comment:
How could this be working? We're defining a variable with the same name of an existing one. Are we 100% sure that `hex.toString()` will use the parameter's value? Also, why do we need `toString()` on something that is already a string?